examples/java/src/main/java/org/apache/beam/examples/complete/TwitterStreamGenerator/TwitterIO.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.examples.complete.TwitterStreamGenerator;
+
+import org.apache.beam.sdk.transforms.Create;
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.values.PBegin;
+import org.apache.beam.sdk.values.PCollection;
+import twitter4j.TwitterStream;
+import twitter4j.*;
+import twitter4j.conf.ConfigurationBuilder;
+
+import java.io.Serializable;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public class TwitterIO {
```

Comment:
can you add Javadoc with an example of how the transform is used?

in Beam, our transforms usually are of this kind:

```
p.apply(TwitterIO.read()  // -> returns a Read transform that has `withX` methods.
     .withKey(...)     // returns a Read transform with the appropriate values set
     .withToken(...)) // etc....
```

so rather than use a Builder and setters, I recommend you add `withX` methods, that you can implement something like this:

```
public Read withX(String x) {
  return this.toBuilder().setX(x).build();
}
```

Reply:
`TwitterIO.read()  // -> returns a Read transform that has `withX` methods.
     .withKey(...)     // returns a Read transform with the appropriate values set
     .withToken(...)`
From this what I understand is that there is a generic read method, and you can read either with Key or with Token. I think reading up on Kafka. Going by KakfaIO as an example: there is a read() in kafkaIO, and we use this to do readFromKafka or 
 readFromKafka -> KafkaIO.<String, String>read()
readAvrosFromKafka -> KafkaIO.<String, String>read()

I have also created a readStandardStream following the same chain of through.

Reply:
Please check now, Is it alright? I have changed the logic.